src/index.html

```diff
@@ -1,13 +1,233 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>THE MET</title>
+    <link rel="icon" href="images/favicon.ico.png">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+
+  <body class="main">
+    <header class="header" id="home">
+      <div class="header__bar">
+        <a href="#">
+          <img src="images/favicon.ico.png" alt="logo">
+        </a>
+        <div class="header__icons">
+          <a href="tel:+1 212-535-7710" class="icon icon--phone">
+            <div class="icon icon--phone--hover">
+              <p>The Met Fifth Avenue</p>
+              <p>+1 212-535--7710</p>
+              <p>The Met Cloisters</p>
+              <p>+1 212-923-3700</p>
+            </div>
+          </a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+      <h1 class="header__title">Welcome to the Met</h1>
+    </header>
+
+    <aside class="menu" id="menu">
+      <div class="menu__bar">
+        <a href="#">
+          <img src="images/favicon.ico.png" alt="Logo">
+        </a>
+        <div class="header__icons">
+          <a href="#" class="icon icon--close"></a>
+        </div>
+      </div>
+      <div class="menu__container">
+        <a href="#">Home</a>
+        <a href="#about-us">About us</a>
+        <a href="#now-on-view">Now on view</a>
+        <a href="#explore-online">Explore online</a>
+        <a href="#contacts">Contacts</a>
+      </div>
+      <div class="menu__contacts">
+        <div class="menu__contacts--address">
+          <a href="https://maps.app.goo.gl/aCcX48tVpvrTQwhD9">The met fifth avenue</a>
+          <a href="tel:+1 212 535-7710">+1 212-535-7710</a>
+        </div>
+        <div class="menu__contacts--address">
+          <a href="https://maps.app.goo.gl/ZetXzAdRkQPHfvTb9">The Met Cloisters</a>
+          <a href="tel:+1 212 923-3700">+1 212-923-3700</a>
+        </div>
+        <a class="menu__contacts--address--call-to-order" href="tel:+1 212 535-7710">Call to order</a>
+      </div>
+    </aside>
+
+    <section class="about-us" id="about-us">
```

Comment:
The styling for this section does not seem to follow the figma design, try to tweak it a little bit:

Yours:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/be2d2e1e-3559-4cc1-8037-1e594ac69e59)

Figma:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/88e9eeb1-093c-4b2c-8b9e-e223837528a4)